src/tls13.c

```diff
@@ -6092,6 +6095,10 @@ static int SendTls13CertificateVerify(WOLFSSL* ssl)
                     goto exit_scv;
             }
 
+            if (args->length > rem) {
+                ERROR_OUT(MP_EXPTMOD_E, exit_scv);
```

Comment:
args->length is the signature length and rem is the amount of space in buffer for message.
Check that the while message and not just the signature will fit.
The error then is a buffer error.